### ISSUE TITLE
Adding Notes to Tables & Chapters

### ISSUE THREAD

**Issue by mgifford** [GSA/open-product-accessibility-template#85]
I'd like to be able to add a note for each chapter like this:

```
chapters:
  success_criteria_level_a:
    notes: "Drupal doesn't make a strong distinction between the front-end & back-end accessibility. Many administration interfaces can be exposed to users in a more interactive site. Generally, this report focuses on the Conformance Level / Remarks and Explanations so that Web comments are about elements that are typically public, while Authoring Tool is typically for authors and administrators. The goal of the authoring interface is to support ATAG 2.0 AA (Part A and B).

The Drupal community strives to keep up with the latest WCAG recommendation."
    criteria:
      - num: "1.1.1"
```

**Comment by dmundra** [comment]
@mgifford I believe notes are supported and show work. See [the following example in the schema](https://github.com/GSA/open-product-accessibility-template/blob/main/schema/opat-0.1.0.json#L46-L49)

Also, there should be displayed in the output. I guess in the examples I added I never added notes for the success_criteria_level_a. Can you try it?

**Comment by mgifford** [comment]
I just added it to this PR to review https://github.com/GSA/open-product-accessibility-template/pull/84

**Comment by dmundra** [comment]
@mgifford I believe notes are working, can you confirm and close this or let me know if you notice any issues?

**Comment by mgifford** [comment]
Seems to be working fine.  The MD output looks great too.
